bench/wtperf/wtperf_config.c

```diff
@@ -1088,7 +1088,6 @@ config_reopen(CONFIG_OPTS *opts)
 
     chunk_cache_cfg_len = strlen(opts->chunk_cache_config);
     req_len = strlen(opts->conn_config) + 1;
-    opts->reopen_connection = true;
```

Comment:
Why are we getting rid of this?

Reply:
This is the bugfix for WT-11323 - it was always marking the config as "needs reopen", which caused the stats to get mismatched. It's set elsewhere, and it was mistakenly left in my last PR after I used it for debugging.